components/ui/TypedWord.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="typed-text__container" :class="{ 'typed-text__cursor': showCursor }">
```

Comment:
We follow a convention in which the main container has a class named after the name of the component in kebab-case so please, could you name this `div` class `typed-word`?